browser/css/w2ui-1.5.rc1.css

```diff
@@ -2943,6 +2943,15 @@ button.w2ui-btn-small:focus:before {
   border: 1px solid transparent;
   background-color: transparent;
 }
+.w2ui-toolbar table.w2ui-button .selected-color-classic {
+  height: 7px;
+  width: 24px;
```

Comment:
Probably this should be 26px, so it is aligned to the top element (since the sibling has 24px width + 1 margin left +1 margin right = 26px)